fava/static/javascript/codemirror-mode-beancount.js

```diff
@@ -38,7 +38,7 @@ CodeMirror.defineSimpleMode('beancount', {
       token: 'whitespace',
     },
     {
-      regex: /[*!&#?%PSTCURM]|txn/,
+      regex: /[*!&?%PSTCURM]|txn/,
```

Comment:
The `beancount`-mode for codemirror does not match tags correctly, like this example:

```
2016-09-15 * "" "Test" #tag1
    Expenses:Test 10.00 USD
    Assets:Test 10.00 USD
```

I found two ways to fix this:

1) Make the change in this line, not matching `#` as a part of the transaction directive, or 
2) moving the regex for matching tags above this line/rule. 

@yagebu What do you think is the best solution?